frontend/src/types/sqlReviewConfig.yaml

```diff
@@ -33,7 +33,7 @@ templateList:
           - key: name_format
             payload:
               type: STRING
-              default: "_delete$"
+              default: ^(_|~)(.+?)_(\d+)_del$
```

Comment:
I actually feel the existing format is more readable. What's the problem with "_del$"

Reply:
Just found seems we're using `(_|~){table}_del` or `(_|~){table}_timestamp_del` for ghost tmp table name. Or maybe using `_del$` here or drop table naming convention is enough?

Reply:
I like _del$